ivy/functional/frontends/tensorflow/__init__.py

```diff
@@ -104,4 +104,8 @@ def check_tensorflow_casting(x1, x2):
 from . import signal
 from . import sparse
 
+
+frontend_array = constant
```

Comment:
I think this is old changes from old commit, please also remove these (repetition with the existing code as well)